code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -254,7 +253,8 @@
 	var/heat_efficency = temperature/(FIRE_MINIMUM_TEMPERATURE_TO_EXIST*100)
 	var/energy_used = heat_efficency*NITRYL_FORMATION_ENERGY
 	ASSERT_GAS(/datum/gas/nitryl,air)
-
+	if ((cached_gases[/datum/gas/oxygen][MOLES] - heat_efficency < 0 )|| (cached_gases[/datum/gas/nitrogen][MOLES] - heat_efficency < 0)) //Shouldn't produce gas from nothing.
```

Comment:
shouldn't these be `<=` checks?

Reply:
and similar checks elsewhere

Reply:
I don't believe so? It's fine if the reaction uses up all the available gas, so going to 0 should be alright.